src/database/migrations/20190901120620-adding_to_user.js

```diff
@@ -0,0 +1,46 @@
+export default {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('Users', 'gender', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'birthday', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'preferredlanguage', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'currency', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'residentialaddress', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'role', {
+      type: Sequelize.STRING,
```

Comment:
User roles should have an enum type. I'm not certain how many roles we can have but you can reach out to @sebaztyn on creating an enum type and setting the values for it